include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -83,10 +83,10 @@ MultigridPreconditionerBase<dim, Number, MultigridNumber>::initialize(
 
   this->initialize_matrix_free_objects();
 
-  this->initialize_transfer_operators();
-
   this->initialize_operators();
 
+  this->initialize_transfer_operators();
```

Comment:
I think the problem is it could happen that we need the transfer operators in order to initialize operators. The concrete example I am thinking of is a velocity or displacement dof-vector that needs to be interpolated to coarser grids. If the present function is called with `initialize_preconditioners = true`, this might lead to a bug because the operators are not initialized completely.